Description: Fix popen-related issues with Emscripten

From: Tom Rothamel <tom@rothamel.us>

warning: unresolved symbol: popen  [emcc]
  missing function: popen  [_popen->abort at runtime]
Forwarded: no
Author: Sylvain Beucler <beuc@beuc.net>
Last-Update: 2018-10-28
---
 Modules/posixmodule.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/Modules/posixmodule.c b/Modules/posixmodule.c
index 7a1a694..7c81f50 100644
--- a/Modules/posixmodule.c
+++ b/Modules/posixmodule.c
@@ -156,8 +156,10 @@ corresponding Unix manual entries for more information on calls.");
 #define HAVE_OPENDIR    1
 #define HAVE_PIPE       1
 #ifndef __rtems__
+#ifndef __EMSCRIPTEN__
 #define HAVE_POPEN      1
 #endif
+#endif
 #define HAVE_SYSTEM     1
 #define HAVE_WAIT       1
 #define HAVE_TTYNAME    1
